SPECS/systemd/systemd-bootstrap.spec

```diff
@@ -37,6 +37,7 @@ Requires:       libgcrypt
 Requires:       lz4
 Requires:       pam
 Requires:       xz
+Requires(pre):  audit-libs
```

Comment:
When I look at the spec now, I think we should go for `Requires(post)` to avoid any confusion - `systemd` doesn't have a `%pre` section, so whatever might be failing during installation would fail in `%post` at the earliest.

Reply:
Makes sense, updated